server/game/cards/13.3-PoS/EddardStark.js

```diff
@@ -0,0 +1,26 @@
+const DrawCard = require('../../drawcard.js');
+
+class EddardStark extends DrawCard {
+	setupCardAbilities(ability) {
+			this.action({
+				condition: () => this.allCharactersHaveStarkAffiliation()
+				title: 'Kill a character'
+				phase: 'dominance'
+				cost: ability.costs.kneelSelf()
+				target: {
+					cardCondition: card => card.location === 'play area' && card.getType() === 'character' && card.getPrintedCost() <= 4
+					gameAction: 'kill'
+				},
+				handler: context => {
+					this.game.addMessage('{0} kneels {1} to kill {2}', this.controller, this, context.target);
```

Comment:
We've started transitioning to a new message format. So while this is correct, we've been trying to implement messages for newer cards like this:
```javascript
message: '{player} kneels {source} to kill {target}',
handler: context => {
  this.game.killCharacter(context.target);
}
```

Reply:
Ah that looks a lot easier to understand when you read the code.